### DISCUSSION TITLE
Image optimiztion for external images

### DISCUSSION THREAD

**Discussion by TpTTT-T** [vercel/next.js#36080] [Help]
About next document,

[https://nextjs.org/docs/basic-features/image-optimization#domains](url)

> To protect your application from malicious users, you must define a list of remote domains that you intend to access this way

What does that malicious users mean?

**Comment by icyJoseph** [comment]
I think you can read the RFC for the Image component, found [here](https://github.com/vercel/next.js/discussions/17141), to learn about this.

> In case of external images you'll need to add the domains (or domain pattern) to an allow list. This is to ensure that malicious users can't hijack the image optimization for their own use
